### PR TITLE
feat: added IDER support

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: src/core/AMTIDER.ts [hunk 1]
```diff
@@ -0,0 +1,256 @@
+import { TypeConverter } from './Converter'
```
Comment (rsdmike) [inline src/core/AMTIDER.ts:1]: missing copyright headers on new files

File: src/core/AMTIDER.ts [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+        return
+      }
+      this.inSequence++
+      this.acc = this.acc.substring(len)
+    }
+  }
+
+  // Private method
+  sendCommand (cmdid: number, data: string = '', completed?: boolean, dma?: number): void {
+    console.log('cmdid :', cmdid, 'data', typeof (data), 'completed :', completed, 'dma :', dma)
```
Comment (rsdmike) [inline src/core/AMTIDER.ts:119]: need to remove all debug statements

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/core/AMTIDER.ts (hunk 1 above)
Comment (madhavilosetty-intel) [reply src/core/AMTIDER.ts:6]: added

File: src/core/AMTIDER.ts (hunk 2 above)
Comment (madhavilosetty-intel) [reply src/core/AMTIDER.ts]: updated

**Review by rsdmike — APPROVED** [review]

**Pull request merged**